### DISCUSSION TITLE
Button to close all Direct Message tabs from non-friends

### DISCUSSION THREAD

**Discussion by chromb** [ppy/osu#18578] [Ideas]
Every so often i open osu!lazer to a LOT of chat windows, making it quite difficult to see recent DMs and such, would it be possible to add a button to close all DMs from non-friends and/or some way to 'pin' or 'favourite' DMs so that they don't close?

**Comment by peppy** [comment]
I'm guessing you also don't use chat on the web, correct?

**Reply by chromb** [reply]
I occasionally use chat on the web, but mostly use IRC / Stable client where possible.
